lib/zone.ts

```diff
@@ -1148,7 +1148,7 @@ const Zone: ZoneType = (function(global: any) {
       const fetchPromise = global['fetch']();
       // ignore output to prevent error;
       fetchPromise.then(() => null, () => null);
-      if (fetchPromise.constructor != NativePromise) {
+      if (fetchPromise.constructor != NativePromise && fetchPromise.constructor != ZoneAwarePromise) {
```

Comment:
Can you break this change into a separate commit. I don't think it has anything to do with NativeScript